remove direct placement (#7400)
pingcap/docs: sql-statements/sql-statement-show-placement-for.md

>
> If you understand the risks, you can enable this experiment feature by executing `SET GLOBAL tidb_enable_alter_placement = 1;`.

> `SHOW PLACEMENT FOR` summarizes all placement options, and presents them in the canonical form for a specific table, database schema, or partition.

Reviewer: placement options, policies, or rules?

Author: - A placement OPTION is something like "FOLLOWERS=4".
- A placement POLICY is a way of grouping options together into a named object which can be assigned to tables.
- Placement RULES previously included both direct placement options and placement policies, but we've since removed direct placement so this term has lost much of its meaning. So it is now usually possible to use RULES and POLICY interchangably, but I think we should keep the terms for now because it is similar to using the terms "RBAC" and "permissions" interchangebly. "Policy" is like the RBAC of Placement.

> The `Scheduling_State` indicates the current progress the Placement Driver (PD) has made in scheduling the placement:

Reviewer: ```suggestion
The statement returns a table in which the `Scheduling_State` column indicates the current progress that the Placement Driver (PD) has made in scheduling the placement:
```

Author: We shouldn't call it a table, it's a result set. Table could be confused with the relational object table.

Reviewer: I've modified my comment. PTAL again. thanks~

Author: LGTM

> * `PENDING`: The PD has not yet started scheduling the placement. This might indicate that that the placement rules are semantically correct, but can not currently be satisfied by the cluster. For example, if `FOLLOWERS=4` but there are only 3 TiKV stores which are candidates for followers.

Reviewer: ```suggestion
* `PENDING`: The PD has not yet started scheduling the placement. This might indicate that the placement rules are semantically correct, but cannot currently be satisfied by the cluster. For example, if `FOLLOWERS=4` but there are only 3 TiKV stores that are candidates for followers.
```

* `INPROGRESS`: The PD is currently scheduling the placement.

> * `COMPLETE`: The PD has successfully scheduled the placement.

Reviewer 2: Ditto.

> * `SCHEDULED`: The PD has successfully scheduled the placement.

Reviewer: I guess these are column fields in the output of `SHOW PLACEMENT FOR`. Would it be better to place these descriptions in the Example section?

Author: I would prefer above, since it applies to all examples. But it's not a strong opinion.
